vignettes/rendering-output.Rmd

```diff
@@ -44,6 +44,12 @@ JSON is not appropriate for every situation, however. If you want your API to re
 
 By default, Plumber serializes objects into JSON via the `jsonlite` R package. However, there are a variety of other serializers that are built in to the package.
 
+You can also pass arguments to certain serializers to modify their behavior like in the example below.
+```r
+#* @serializer json list(na="string")
+```
+See [here](../reference/serializers.html) for possible arguments.
```

Comment:
```suggestion
See the [Serialization](../reference/serializers.html) article for details.
```

Reply:
Is CRAN going to complain here too?

Reply:
Dang. Yes. Missed it. Updating the comment above